.github/workflows/conventional-commit.yml

```
@@ -36,6 +36,98 @@
+            echo "Found ticket ID in branch: $ticket_id"
+            
+            # Check if PR title already has the Linear ticket format
+            if [[ "$pr_title" =~ fixes\ [A-Z]+-[0-9]+$ ]]; then
+              echo "PR title already contains Linear ticket ID"
+              echo "needs_update=false" >> $GITHUB_OUTPUT
+              echo "linear_ticket_valid=true" >> $GITHUB_OUTPUT
+            else
+              # Auto-append the Linear ticket ID
+              new_title="$pr_title fixes $ticket_id"
+              echo "Auto-fixing PR title to: $new_title"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)